docs/experimental/lakectl_local_posix.md

```diff
@@ -0,0 +1,99 @@
+---
+title: lakectl local file permissions tracking
+description: An experimental feature that allows tracking file ownership and permissions when using lakectl local
+parent: none
+---
+
+# lakectl local file permission tracking
+{: .d-inline-block }
+Experimental
+{: .label .label-green }
+
+This experimental feature is used to:
+* Support mode preservation for files and folders managed via lakectl local **for POSIX compliant filesystems only**
+* Support user and group preservation for files and folders managed via lakectl **local for POSIX compliant filesystems only**
+
+
+{: .note}
+> This feature is currently supported for Unix based Operating Systems only
```

Comment:
It also works on Linux.
```suggestion
> This feature is currently supported for POSIX based Operating Systems only
```

Reply:
Yes but Linux is Unix based *and* we don't support windows which may support POSIX FS